src/components/Navigation.jsx

```diff
@@ -10,11 +10,11 @@ import { FiSettings } from 'react-icons/fi';
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/226111339-2e1fec37-4980-455e-a465-474450bfd868.png)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages capitalize to make your pull request professional :blush:

Reply:
Thanks, will keep this in mind.